gtsfm/utils/metrics.py

```diff
@@ -531,7 +531,8 @@ def pose_auc(errors: np.ndarray, thresholds: Sequence[float], save_plot: bool =
             plt.ylabel("Recall")
             plt.xlabel("Pose Error (deg.)")
             uuid = datetime.datetime.utcnow().strftime("%Y_%m_%d_%H_%M_%S_%f")
-            save_fpath = f"{uuid}_recall_vs_pose_error_curve_auc.jpg"
+            Path("pose_auc").mkdir(parents=True, exist_ok=True)
+            save_fpath = Path("pose_auc", f"{uuid}_recall_vs_pose_error_curve_auc_{t}_deg_threshold.jpg")
```

Comment:
Should this go under results or plots directory?

Reply:
yes, I frequently use the output_root flag.

Reply:
Fixed now, thanks.